data/companies.yml

```diff
@@ -472,6 +472,9 @@ users:
   - name: "US Air Force"
     logo: "/logos/usaf.svg"
     url: "https://www.af.mil/"
+  - name: "T-Mobile US"
```

Comment:
This should be added alphabetically within the section.

Reply:
Just an FYI, but you can look at https://deploy-preview-10777--preliminary-istio.netlify.app/latest/about/case-studies (or click on Details to the right of the deploy/netlify test and go through the site). You can see what will be deployed, and if you scroll down you will see T-mobile between the US Air Force and Walmart.

Reply:
Derp....sorry about that. Pushed a commit to fix the ordering

Reply:
Hi Joe,

T-Mobile is already listed, just further up the list in the Case Studies section, highlighting your Istiocon talk from last year.

The wall of logos is "and all these others". 

Would you prefer we updated the logo to the one you've provided, as I see they are different?

Reply:
Oh, I had not noticed it was up above. Thank you for catching that. I'll just close this PR. Sorry for missing that.